hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -124,7 +121,7 @@ def lambda_handler(event, context):
                         str(signal_id), str(privacy_group), str(metadata["source"])
                     )
                     for tag in metadata["tags"].get(privacy_group, []):
-                        banked_signal.classifications.append(tag)
+                        banked_signal.add_classification(tag)
```

Comment:
this call to `add_classification()` does the work that was removed from action_evaluator.py.